src/styles/cards.scss

```diff
@@ -0,0 +1,100 @@
+@import "./card-stars";
+
+.card {
+  font-size: 10px;
+  display: flex;
+  flex-direction: column;
+  padding: 32px 16px 16px;
+  border-radius: 5px;
+  border: 1px solid $border-color;
+  background: $color-white;
+  transition-duration: 300ms;
```

Comment:
The transition-duration property needs a property to transition, for example 'transition-duration: transform 300ms'.